package.json

```diff
@@ -16,7 +16,8 @@
     "turbo-prebuilt": "node turbow.js",
     "docs": "pnpm -- turbo run dev --filter=docs --no-cache",
     "run-example": "./scripts/run-example.sh",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test": "pnpm -- turbo run test"
```

Comment:
Following the pattern, but if we had global turbo in Github CI, we wouldn't need to use `pnpm -- turbo`